lib/jsonrpccxx/client.hpp

```diff
@@ -1,92 +1,126 @@
 #pragma once
-#include "common.hpp"
-#include "iclientconnector.hpp"
 #include <exception>
 #include <nlohmann/json.hpp>
 #include <string>
 #include <variant>
 
+#include "common.hpp"
+#include "iclientconnector.hpp"
+
 namespace jsonrpccxx {
-  enum class version { v1, v2 };
-  typedef std::vector<json> positional_parameter;
-  typedef std::map<std::string, json> named_parameter;
-  typedef std::variant<int, std::string> id_type;
 
-  struct JsonRpcResponse {
-    id_type id;
-    json result;
-  };
+enum class version { v1, v2 };
+typedef std::vector<json> positional_parameter;
+typedef std::map<std::string, json> named_parameter;
+typedef std::variant<int, std::string> id_type;
 
-  class JsonRpcClient {
-  public:
-    JsonRpcClient(IClientConnector &connector, version v) : connector(connector), v(v) {}
-    virtual ~JsonRpcClient() = default;
+struct JsonRpcResponse {
+  id_type id;
+  json result;
+};
 
-    template <typename T>
-    T CallMethod(const id_type &id, const std::string &name) { return call_method(id, name, json::object()).result.get<T>(); }
-     template <typename T>
-    T CallMethod(const id_type &id, const std::string &name, const positional_parameter &params) { return call_method(id, name, params).result.get<T>(); }
-    template <typename T>
-    T CallMethodNamed(const id_type &id, const std::string &name, const named_parameter &params = {}) { return call_method(id, name, params).result.get<T>(); }
+class JsonRpcClient {
+ public:
+  JsonRpcClient(IClientConnector &connector, version v)
+      : connector(connector), v(v) {}
+  virtual ~JsonRpcClient() = default;
 
-    void CallNotification(const std::string &name, const positional_parameter &params = {}) { call_notification(name, params); }
-    void CallNotificationNamed(const std::string &name, const named_parameter &params = {}) { call_notification(name, params); }
+  template <typename T>
```

Comment:
should not we call methods with NO capital letters?

Reply:
It's third-party library - this is how they wrote their functions. I just added another one with the same style to not create more chaos.